doc/developers/developing_scikit.rst

```diff
@@ -0,0 +1,671 @@
+.. _developing_scikit:
+
+==================================
+Developing scikit-learn estimators
+==================================
+
```

Comment:
Insert `.. currentmodule:: sklearn`

Reply:
@jnotham I am not sure why but I am getting build failures with '.. currentmodule:: sklearn'. But once I removed it all the failures are gone.